src/domain_bridge/domain_bridge.cpp

```diff
@@ -402,9 +402,23 @@ class DomainBridgeImpl
         } else {
           qos.lifespan(qos_match.qos.lifespan());
         }
-
-        qos.liveliness(qos_match.qos.liveliness());
-        qos.liveliness_lease_duration(qos_match.qos.liveliness_lease_duration());
+        if (qos_options.liveliness()) {
+          qos.liveliness(*qos_options.liveliness());
+        } else {
+          qos.liveliness(qos_match.qos.liveliness());
+        }
+        if (qos_options.liveliness_lease_duration()) {
+          const auto liveliness_lease_duration_ns = *qos_options.liveliness_lease_duration();
```

Comment:
nit : Alignment

Reply:
what do you mean exactly?
This one looks fine, it's one line.

Uncrustify is passing.